website/templates/share_search.mako

```diff
@@ -6,4 +6,16 @@
 
 <%def name="javascript_bottom()">
     <script src=${"/static/public/js/share-search-page.js" | webpack_asset}></script>
+
+<script type="text/x-mathjax-config">
+   MathJax.Hub.Config({
+       tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']], processEscapes: true}
+    });
+</script>
+
+<script type="text/javascript"
+    src="/static/vendor/bower_components/MathJax/unpacked/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+</script>
```

Comment:
Let's indent these the same level as the `<script>` that's already in this mako block.